### ISSUE TITLE
Document the "Unsign xcodebuild too?" option.

### ISSUE THREAD

**Issue by jklausa** [inket/update_xcode_plugins#8]
Your awesome gem is helpful, but the prompt about unsigning `xcodebuild` is not :)

Just prompting the question without giving any context what potential consequences are there is not very helpful. I'd be happy to submit a PR if you could explain to me in plain english what happens if I let it/prevent it from unsigning it.

Thanks a lot for this gem!

**Comment by inket** [comment]
Hi!

If `xcodebuild` is signed, running it from the command-line (whether for simply building, running tests or through carthage build) will print warnings about not being able to load plugins because of library validation.

Unsigning it would hide those warnings, and technically also allow for plugins that modify `xcodebuild`'s functions.

The binary that gets unsigned is `/Applications/Xcode-unsigned.app/Contents/Developer/usr/bin/xcodebuild` and you would need to set the toolchain path to Xcode-unsigned to be able to run it:

```
$ sudo xcode-select -s /Applications/Xcode-unsigned.app/Contents/Developer
```

**Comment by inket** [comment]
Added info to README about this and linked to this issue. Closing this now. Thanks for bringing it up!
